src/cli/subcommand/install.rs

```diff
@@ -4,10 +4,12 @@ use std::{
 };
 
 use crate::{
-    action::ActionState, cli::is_root, plan::RECEIPT_LOCATION, BuiltinPlanner, InstallPlan, Planner,
+    action::ActionState, cli::is_root, cli::signal_channel, plan::RECEIPT_LOCATION, BuiltinPlanner,
+    InstallPlan, Planner,
 };
 use clap::{ArgAction, Parser};
 use eyre::{eyre, WrapErr};
+use tokio_util::sync::CancellationToken;
```

Comment:
This looks unused to me, but I could be wrong.

Reply:
It sure is!